added STM zones file (#5160)
centreon/centreon-documentation: cxm/configuration/user-journey/stm-zone.md

---
id: stm-zones
title: Monitoring non-public user journeys (closed beta)
---

> A private Synthetic Monitoring Zone (STM) allows you to monitor your user journeys on internal domains or networks specific to your organization using a probe deployed inside your infrastructure.

Reviewer: entre parenthèses je mettrais plutôt (STM **Zone**)

> A private Synthetic Monitoring Zone (STM) allows you to monitor your user journeys on internal domains or networks specific to your organization using a probe deployed inside your infrastructure.

Author: ```suggestion
A private Synthetic Monitoring Zone (STM Zone) allows you to monitor your user journeys on internal domains or networks specific to your organization using a probe deployed inside your infrastructure.
```

- A user journey configured on the internal application to monitor.

## Step 1: Create a new STM zone

1. From the **Global View**, open the site selector in the top-left corner and select your organization’s page.

![image](../../assets/configuration/user-journey/organization-page.png)

2. On your organization’s configuration page, click the **Synthetic Monitoring Zones** tab.

3. Click **+ New Synthetic Monitoring Zone**. Give the zone a meaningful name (e.g., Paris Office) then click **+ Create**.

Your new zone now appears in the list.

## Step 2: Associate a probe to the STM zone

1. Click **Associate a probe** to the right of your zone. A window opens with 2 Docker commands:

2. Use the first command to log into the Centreon Docker registry with [the credentials provided by Centreon](#prerequisites)

```shell
docker login docker.centreon.com/centreon-dem-beta
```

## Step 3: Create and launch the probe

1. To create and launch the probe, execute the second command you obtained at [step 2](#step-2-associate-a-probe-to-an-stm).

> 2. Refresh the page: once launched, the probe is automatically saved and appears to the right of the associated zone in the **Synthetic Monitoring Zones** liste.

Reviewer: "list" (sans E final)

> 2. Refresh the page: once launched, the probe is automatically saved and appears to the right of the associated zone in the **Synthetic Monitoring Zones** liste.

Author: ```suggestion
2. Refresh the page: once launched, the probe is automatically saved and appears to the right of the associated zone in the **Synthetic Monitoring Zones** list.
```

## Step 4: Associate the zone with a user journey

1. Go to **Configuration** and select the **User Journeys** tab. 

2. On the journey you want to run from your private zone, click on the three dots on the right, then click on **Advanced**.

> 3. In the **Advanced configuration** window, scroll down to the **Synthetic Monitoring Zones** section. Your private zone appears under **Private Zones**. Select it and click **Save**

Reviewer: manque le point final

> 3. In the **Advanced configuration** window, scroll down to the **Synthetic Monitoring Zones** section. Your private zone appears under **Private Zones**. Select it and click **Save**

Author: ```suggestion
3. In the **Advanced configuration** window, scroll down to the **Synthetic Monitoring Zones** section. Your private zone appears under **Private Zones**. Select it and click **Save**.
```

---
id: stm-zones
title: Monitoring non-public user journeys (closed beta)
---

A private Synthetic Monitoring Zone (STM) allows you to monitor your user journeys on internal domains or networks specific to your organization using a probe deployed inside your infrastructure.

> ## Prerequisites:

Reviewer 2: ```suggestion
## Prerequisites
```

- A machine inside your infrastructure to host the probe. The machine must be able to access the application you will monitor.
- The Docker credentials provided by Centreon. The credentials are sent by Centreon using a secure Keeper link. Save these credentials in your own safe.

> - A **user journey** configured on the internal application to monitor.

Reviewer 2: ```suggestion
- A user journey configured on the internal application to monitor.
```

## Step 1: Create a new STM zone

> - From the **Global View**, open the site selector at the top left and open your organization’s page.

Reviewer 2: ```suggestion
- From the **Global View**, open the site selector in the top-left corner and select your organization’s page.
```

> - From the **Global View**, open the site selector at the top left and open your organization’s page.

Reviewer 2: numbered steps

![image](../../assets/configuration/user-journey/organization-page.png)

- On your organization’s configuration page, click the **Synthetic Monitoring Zones** tab.

> - Click **+ New Synthetic Monitoring Zone**. Give it a meaningful name (e.g., Paris Office) then click **+ Create**.

Reviewer 2: ```suggestion
- Click **+ New Synthetic Monitoring Zone**. Give the zone a meaningful name (e.g., **Paris Office**) then click **+ Create**.
```

> Your new zone now appears in the list.

Reviewer 2: ```suggestion

   Your new zone now appears in the list.
```

> ## Step 2: Associate a probe to an STM

Reviewer 2: ```suggestion
## Step 2: Associate a probe to the STM zone
```

- Click **Associate a probe** to the right of your zone.

> A window opens with 2 Docker commands:

Reviewer 2: ```suggestion
1. Click **Associate a probe** to the right of your zone. A window opens with 2 Docker commands.
```

> - Use the first command to log into the Centreon Docker registry with [the credentials provided by Centreon](#prerequisites)

Reviewer 2: ```suggestion
2. Use the first command to log into the Centreon Docker registry with [the credentials provided by Centreon](#prerequisites). Enter your username and password when prompted.
```

```shell
docker login docker.centreon.com/centreon-dem-beta
```

> Enter your username and password when prompted.

Reviewer 2: ```suggestion
```

> - The second command is used to create and launch the probe (see [step 3](#step-3-create-and-launch-the-probe))

Reviewer 2: ```suggestion
```

## Step 3: Create and launch the probe

> To create and launch the probe, execute the second command you obtained at [step 2](#step-2-associate-a-probe-to-an-stm).

Reviewer 2: ```suggestion
1. To create and launch the probe, execute the second command you obtained at [step 2](#step-2-associate-a-probe-to-an-stm).
```

> Refresh the page: once launched, the probe is automatically saved and appears to the right of the associated zone in the **Synthetic Monitoring Zones** liste.

Reviewer 2: ```suggestion
2. Refresh the page: once launched, the probe is automatically saved and appears to the right of the associated zone in the **Probes** column.
```

## Step 4: Associate the zone with a user journey

> 1. Go to **Configuration** > **User Journeys** tab. On the journey you want to run from your private zone, click on the three dots on the right, then click on **Advanced**.

Reviewer 2: ```suggestion
1. Go to **Configuration** and select the **User Journeys** tab.
2. On the journey you want to run from your private zone, click on the three dots on the right, then click on **Advanced**.
```

> 2. In the **Advanced configuration** window, scroll down to the **Synthetic Monitoring Zones** section. Your private zone appears under **Private Zones**. Select it.

Reviewer 2: ```suggestion
3. In the **Advanced configuration** window, scroll down to the **Synthetic Monitoring Zones** section. Your private zone appears under **Private Zones**. Select it and click **Save**.
```

> Click **Save**

Reviewer 2: ```suggestion
```

> 3. After a few seconds, the probe will have executed its first check and your internal journey monitoring will be operational! You can analyze it as you would for a regular [user journey](../../how-to-articles/user-journey-screen.md).

Reviewer 2: ```suggestion
After a short while, the probe will have executed its first check and your internal journey monitoring will be operational. You can analyze it just like a regular [user journey](../../how-to-articles/user-journey-screen.md).
```

## Troubleshooting

### This domain name is not allowed for your website

> If you encounter this message when trying to set a navigation action to a link, it means the domain you are trying to navigate to has not been authorized by Centreon yet.

Reviewer 2: unclear

![image](../../assets/configuration/user-journey/organization-page.png)

## Step 2: Access the STM Zones tab

On your organization’s configuration page, click the “Synthetic Monitoring Zones” tab.

## Step 3: Create a new STM Zone

Click **+ New Synthetic Monitoring Zone**, give it a meaningful name (e.g., Paris Office), then click **+ Create**.
Your new zone now appears in the list.

## Step 4: Associate a sensor

Click **Associate a sensor** using the button in the top-right corner or directly on the line in your zone.

A modal window opens with two Docker commands:

the first one is used to log in to the Centreon registry (see Step 5)

the second one is used to launch the sensor (see Step 6)

## Step 5: Obtain the Docker credentials from Keeper

The sensor image is stored in a private registry. The credentials must be shared via a secure Keeper link.

Contact Centreon to receive the Keeper link. Save the credentials in your own vault.

Once you have obtained the credentials, log in to the Docker registry:

'''shell
docker login docker.centreon.com/centreon-dem-beta
Username: <entrez votre nom utilisateur>

> Password: <entrez votre mot de passe>

Author: ```suggestion
Username: \<username\>
Password: \<password\>
```

![image](../../assets/configuration/user-journey/organization-page.png)

## Step 2: Access the STM Zones tab

On your organization’s configuration page, click the **Synthetic Monitoring Zones** tab.

## Step 3: Create a new STM zone

Click **+ New Synthetic Monitoring Zone**, give it a meaningful name (e.g., Paris Office), then click **+ Create**.
Your new zone now appears in the list.

## Step 4: Associate a probe

Click **Associate a probe** using the button in the top-right corner or directly on the line in your zone.

A window opens with two Docker commands:

the first one is used to log in to the Centreon registry (see [Step 5](#step-5-obtain-the-docker-credentials-from-keeper))

the second one is used to launch the probe (see [Step 6](#step-6-launching-the-probe))

## Step 5: Obtain the Docker credentials from Keeper

The credentials are sent to you by Centreon via a secure Keeper link.

Contact Centreon to receive the Keeper link. Save the credentials in your own vault.

Once you have obtained the credentials, log in to the Docker registry:

'''shell
docker login docker.centreon.com/centreon-dem-beta
Username: \<username>
Password: \<password>
'''

> ## Step 6: Launching the probe

Reviewer 3: c'est le seul titre qui n'est pas un verbe à l'infinitif

Once launched, the probe is automatically saved and appears in the STM Zones list.

## Step 7: Associate the zone with a user journey

Go to the **User Journeys** tab on your site. On the journey you want to run from your private zone, click on the three dots on the right, then click on **Advanced**.

In the **Advanced configuration** window, scroll down to the **Synthetic Monitoring Zones** section. Your private zone appears under Private Zones. Select it.
Click Save

> Your internal journey monitoring is now in place! You can analize it as a you would for a regular [user journey](../how-to-articles/user-journey-screen.md).

Reviewer 3: analize => analyze

> Your internal journey monitoring is now in place! You can analize it as a you would for a regular [user journey](../how-to-articles/user-journey-screen.md).

Reviewer 3: as a you would => as you would

---
id: stm-zones
title: Monitoring non-public user journeys
---

> A private Synthetic Monitoring Zone (STM) allows you to monitor your user journeys on internal domains or networks specific to your organisation using a probe deployed inside your infrastructure.

Reviewer 3: organisation => organization

> A private Synthetic Monitoring Zone (STM) allows you to monitor your user journeys on internal domains or networks specific to your organisation using a probe deployed inside your infrastructure.

Author: ```suggestion
A private Synthetic Monitoring Zone (STM) allows you to monitor your user journeys on internal domains or networks specific to your organization using a probe deployed inside your infrastructure.
```

## Step 1: Open the organization page

From the **Global View**, open the site selector at the top left and open your organization’s page.

![image](../../assets/configuration/user-journey/organization-page.png)

> 

Author: ```suggestion
```

- On your organization’s configuration page, click the **Synthetic Monitoring Zones** tab.

## Step 3: Create a new STM zone

- Click **+ New Synthetic Monitoring Zone**, give it a meaningful name (e.g., Paris Office), then click **+ Create**.
Your new zone now appears in the list.

## Step 4: Associate a probe to an STM

- Click **Associate a probe** to the right of your zone. You can also click in the upper-right corner in which case, make sure the right zone to select the right zone in the new window.

A window opens with two Docker commands:

- the first one is used to log in to the Centreon Docker registry (see [step 5](#step-5-obtain-the-docker-credentials-from-keeper)),

> - the second one is used to launch the probe (see [step 6](#step-6-launch-the-probe)).

Author: ```suggestion
- the second one is used to create and launch the probe (see [step 6](#step-6-launch-the-probe)).
```

the second one is used to launch the probe (see [step 6](#step-6-launching-the-probe))

## Step 5: Obtain the Docker credentials from Keeper

1. The credentials are sent to you by Centreon via a secure Keeper link.

> 2. Contact Centreon to receive the Keeper link. Save the credentials in your own vault.

Reviewer 3: C'est étonnant de dire. "les creds sont envoyé par centreon" puis, "contactez Centreon pour recevoir les creds"

> 3. Once you have obtained the credentials, log in to the Docker registry:

Author: ```suggestion
The credentials are sent to you by Centreon via a secure Keeper link.

1. Contact Centreon to receive the Keeper link. Save the credentials in your own vault.

2. Once you have obtained the credentials, log in to the Docker registry:
```

'''shell
docker login docker.centreon.com/centreon-dem-beta
Username: <entrez votre nom utilisateur>

> Password: <entrez votre mot de passe>

Author: ```suggestion
Username: \<username>
Password: \<password>
```

'''

> ## Step 6: Launching the probe

Author: ```suggestion
## Step 6: Launch the probe
```

> To launch the probe, execute the second command you obtained at [step 4](#step-4-associate-a-probe), if you had previously closed the window, you can simply open it again.

Author: ```suggestion
## Step 6: Create and launch the probe

To create and launch the probe, execute the second command you obtained at [step 4](#step-4-associate-a-probe), if you had previously closed the window, you can simply open it again.
```

Once launched, the probe is automatically saved and appears in the STM Zones list.

## Step 7: Associate the zone with a user journey

> 1. Go to the **User Journeys** tab on your site. On the journey you want to run from your private zone, click on the three dots on the right, then click on **Advanced**.

Author: ```suggestion
1. Go to **Configuration** > **User Journeys** tab. On the journey you want to run from your private zone, click on the three dots on the right, then click on **Advanced**.
```

2. In the **Advanced configuration** window, scroll down to the **Synthetic Monitoring Zones** section. Your private zone appears under Private Zones. Select it.
Click Save

> 3. Your internal journey monitoring is now in place! You can analize it as a you would for a regular [user journey](../how-to-articles/user-journey-screen.md).

Author: ```suggestion
3. Your internal journey monitoring is now in place! You can analyze it as you would for a regular [user journey](../how-to-articles/user-journey-screen.md).
```
